src/remarkPlugin.js

```diff
@@ -155,7 +155,11 @@ function remarkPlugin(options = {}) {
     //    time, change the original code fence Markdown node to an HTML node and set
     //    its value to the HTML rendering contained in the GraphQL node.
 
+    let numCodeOccurrences = 0;
```

Comment:
It would be nicer to just be able to ask `codeNodeRegistry.isEmpty()` or something